v62000/u_inref/gtm8086.csh

```diff
@@ -153,7 +153,7 @@ if ($?test_replic) then
 		# getoper will capture syslog messages generated since starting the jnlswitchretry error
 		$gtm_tst/com/getoper.csh "$jnlswitchretry_time"
 		# Confirm there is no PREVJNLLINKCUT error in the output
-		$grep -e "PREVJNLLINKCUT" syslog.txt
+		$grep -e "$PWD" syslog.txt | $grep -e "PREVJNLLINKCUT"
```

Comment:
It is better to do the PREVJNLLINKCUT grep first as that will produce fewer lines for the second grep to process.
Also, why do we need the -e flag?